### DISCUSSION TITLE
Search and display all found results, and highlight all

### DISCUSSION THREAD

**Discussion by styw1234** [sumatrapdfreader/sumatrapdf#2906] [Ideas]
it is suggested that after searching, all found results should be displayed and highlighted.

**Comment by GitHubRulesOK** [comment]
highlight all
 is usually driven by javascript as its beyond the standard find first (mutool javascript can permanently highlight all matching strings)
 for how its done in firefox.js but not in binary viewer ChromiumEdge see examples at https://stackoverflow.com/a/73299830/10802527

**Reply by icyhoty2k** [reply]
how to hichlight select word(text) occurances automatically?

**Reply by styw1234** [reply]
use shortcut ‘a’

**Reply by styw1234** [reply]
> 

can sumatra hightlight all results which are searched

**Comment by fietronic** [comment]
make it happen captain
